metrics/cli/lib/services/firebase/firebase_service.dart

```diff
@@ -17,11 +17,26 @@ abstract class FirebaseService extends InfoService {
 
   /// Deploys the given Firebase hosting [target] from the given [appPath]
   /// to the Firebase project with the given [projectId] hosting.
-  Future<void> deployHosting(String projectId, String target, String appPath);
+  ///
+  /// The [authToken] is an optional parameter for the direct
+  /// Firebase authentication.
```

Comment:
:readable:
```suggestion
  /// Authenticates the deployment process using the given [authToken] if it is not null. Otherwise, authenticates using the global Firebase user. 
```
What do you think? Please, apply this suggestion to all similar situations.